spec/index.html

```diff
@@ -259,28 +255,100 @@ <h2>A Canonical form of N-Quads</h2>
       and <code>graphLabel</code> if present, MUST be a single space, 
       (<code>U+0020</code>).  All other locations that allow 
       white space MUST be empty.</li>
-    <li>There MUST be no comments.</li>
-    <li><code>HEX</code> MUST use only uppercase letters (<code>[A-F]</code>).</li>
-    <li>Characters MUST NOT be represented by <code>UCHAR</code>.</li>
+    <li><a data-cite="RDF12-CONCEPTS#literal">Literals</a> with the
+      datatype <code>http://www.w3.org/2001/XMLSchema#string</code>
+      MUST NOT use the datatype IRI part of the <a href="#grammar-production-literal">literal</a>,
+      and are represented using only <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>.
+    </li>
+    <li>There MUST be no comments or <a>blank lines</a>.</li>
```

Comment:
Strictly this is not necessary because it is consequence of the other rules for white space and EOL. As such, move to the end of the bulleted list.